aptos-move/framework/aptos-stdlib/sources/cryptography/algebra_bls12381.move

```diff
@@ -17,28 +220,525 @@ module aptos_std::algebra_bls12381 {
     /// NOTE: the same scheme is also used in other implementations (e.g., ark-bls12-381-0.4.0, blst-0.3.7).
     struct FrFormatLsb {}
 
+    /// A serialization scheme for `Fr` elements,
+    /// where an element is represented by a byte array `b[]` of size 32 with the most significant byte coming first.
+    ///
+    /// NOTE: the same scheme is also used in other implementations (e.g., ark-bls12-381-0.4.0, blst-0.3.7).
+    struct FrFormatMsb {}
+
+    //
+    // (Marker types + serialization formats end here.)
+    // Hash-to-structure suites begin.
+    //
+
+    /// The hash-to-curve suite `BLS12381G1_XMD:SHA-256_SSWU_RO_` that hashes a byte array into `G1Affine` elements.
+    ///
+    /// Full specification is defined in https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-hash-to-curve-16#name-bls12-381-g1.
+    struct H2SSuiteBls12381g1XmdSha256SswuRo {}
```

Comment:
Consider renaming the `H2SSuiteBls12381` to just `Hash` (remember that bls12381 is already part of the module name). Plus, `Hash` will be easier to find in autocomplete.

Reply:
Fixed.